src/components/Counter/Counter.styles.jsx

```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'aphrodite';
+
+const CounterStyles = StyleSheet.create({
+  button: {
+    margin: '10px',
+    backgroundColor: '#ff0000',
+    color: '#fff',
+    fontSize: '16px',
+    width: '100px',
+    height: '40px',
```

Comment:
By default, constant values are coerced into `px` units.

So
```css
height: '40px'
```

is same as...

```css
height: 40
```